src/Forwarder.cpp

```diff
@@ -39,6 +39,14 @@ std::vector<char> getHostname() {
 #endif
 
 #include "CURLReporter.h"
+#include "schemas/f142/f142.cpp"
+
+namespace {
+void registerSchemas() {
```

Comment:
Considering that we only have one module in the forwarder (I should REALLY finish that pull request for the chopper module), I do not mind putting this as C++ code. However, I am still interested in hearing if you have any reasoning to do this as it has been working ore or less as intended.

Reply:
After my CMake changes the module was failing to be registered and it was not obvious to me how the magic it was using before was working, I thought that was reason enough to make it explicit.

Reply:
Ok, that is a good enough reason for me.